databricks/koalas/functions.py

```diff
@@ -0,0 +1,70 @@
+#
```

Comment:
Maybe it's best to add module docs that this file is supposed to have PySpark functions. We could name it `functions.py` to something like `spark_functions.py` too ...